README.md

```diff
@@ -7,6 +7,17 @@ providing an easy way to create import / export dataflow.
 
 [![Coverage Status](https://coveralls.io/repos/github/code-rhapsodie/dataflow-bundle/badge.svg)](https://coveralls.io/github/code-rhapsodie/dataflow-bundle)
 
+Dataflow uses a linear generic workflow in three parts (one reader, any number of steps, one or more writers).
+
+The reader can read data from anywhere and return data row by row. Each step processes the current row data. 
+The steps are executed in the order in which they are added.
+And finally, one or more writers save the row anywhere you want.
```

Comment:
It does not really make sense to say `finally` if the is not a first step.
Change this into something like : first the reader ... then ... finally ...
Or remove the `finally`